packages/trader/src/Constants/contract.tsx

```diff
@@ -0,0 +1,304 @@
+import React from 'react';
+import { localize, Localize } from '@deriv/translations';
+
+export type TContractsKeys = keyof ReturnType<typeof getContractConfig>;
```

Comment:
@farzin-deriv Is it possible to leave export? Because I need it in test (lines 78, 81, 90)  to pass a value, which is not expected by function (in order to check if it returns empty string / other default value in that cases)